.githooks/commit-msg

```diff
@@ -7,12 +7,12 @@ TEMPORARY_FILE_PATH=$1
 COMMIT_MSG=`head -n1 $TEMPORARY_FILE_PATH`
 
 ## init regex to match commit-message format
-COMMIT_MSG_REGEX="\I#[0-9]+|Merge branch\s.*"
+COMMIT_MSG_REGEX="\I-#[0-9]+|Merge branch\s.*"
 
 ## checking commit-message format
 if ! [[ $COMMIT_MSG =~ $COMMIT_MSG_REGEX ]]; then
     echo -e "Your commit-message format is not valid:\n$COMMIT_MSG\n"
-    echo "Did you add an 'I' before the #issueNumber? Valid format example:"
+    echo "Did you add an 'I-' before the #issueNumber? Valid format example:"
     echo "I#issueNumber My commit message"
```

Comment:
Please consider adding an `I-` in this example line as well

Reply:
Please consider adding an I- in this example line as well